src/Data/List/Relation/Binary/Permutation/Propositional/Properties.agda

```diff
@@ -354,3 +359,17 @@ module _ {ℓ} {R : Rel A ℓ} (R? : Decidable R) where
     (x ∷ xs) ++ y ∷ ys       ≡˘⟨ Lₚ.++-assoc [ x ] xs (y ∷ ys) ⟩
     x ∷ xs ++ y ∷ ys         ∎
     where open PermutationReasoning
+
+------------------------------------------------------------------------
+-- product
+
+productPreserves↭⇒≡ : product Preserves _↭_ ⟶ _≡_
```

Comment:
The standard name is `product-pres-↭-≡` but I think it's safe to call this `product-↭`?